utils/_context/_scenarios.py

```diff
@@ -465,6 +464,28 @@ class scenarios:
         backend_interface_timeout=5,
     )
 
+    # OpenTelemetry tracing end-to-end scenarios
+    otel_tracing_e2e_w3c = EndToEndScenario(
+        "OTEL_TRACING_E2E_W3C",
+        weblog_env={"DD_API_KEY": os.environ.get("DD_API_KEY"), "DD_SITE": os.environ.get("DD_SITE"),},
```

Comment:
You don't need these, since they are already set.

Reply:
It's actually needed - we set dd api key and dd site for Agent containers but not Weblog containers.